build.gradle

```diff
@@ -1,14 +1,18 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    // Required for sdk version retrieval to work as of Android Studio 4.1.0: https://issuetracker.google.com/issues/158695880
+    project.ext.set("sdk_version_code", 42)
+    project.ext.set("sdk_version_name", "0.5.13")
```

Comment:
please instead move it to `gradle.properties` or root-level `ext { … }` outside `buildscript`. Here is more detailed explanation:
<img width="819" alt="image" src="https://github.com/user-attachments/assets/547911ff-f194-447f-99a6-3b2a049167ae" />